src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -229,15 +216,8 @@ public static double sumLog10(final double[] log10values) {
     public static int[] vectorDiff(final int[]x, final int[] y) {
```

Comment:
this whole method seems to be the same as `MathArrays.ebeSubtract`. I think we should just delegate